extensions/wikia/AdEngine/js/SlotTracker.js

```diff
@@ -3,8 +3,9 @@
 
 define('ext.wikia.adEngine.slotTracker', [
 	'ext.wikia.adEngine.adContext',
-	'ext.wikia.adEngine.adTracker'
-], function (adContext, adTracker) {
+	'ext.wikia.adEngine.adTracker',
+	'ext.wikia.adEngine.adLogicPageParams'
+], function (adContext, adTracker, pageParamsLogic) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "pageParamsLogic". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)